src/observable/BoundCallbackObservable.ts

```diff
@@ -15,6 +15,7 @@ export class BoundCallbackObservable<T> extends Observable<T> {
   subject: AsyncSubject<T>;
 
   /* tslint:disable:max-line-length */
+  static create(callbackFunc: (callback: () => any) => any, selector?: void, scheduler?: IScheduler): () => Observable<void>;
```

Comment:
It's not related to this PR, but I just noticed `bindNodeCallback` doesn't have corresponding overloading signatures. Maybe we need this? /cc @david-driscoll